src/domains/chart/components/chart-container.tsx

```diff
@@ -10,23 +10,23 @@ export type Props = {
   attributes: Attributes
   // warning! this is not the same as chartId in old dashboard
   // here, the chartID must be unique across all agents
-  chartID: string
+  chartUuid: string
```

Comment:
Minor note we use the ID convention nomatter if its a uuid or other. Not important. just noting...

Reply:
so the problem here is that for the Agent, chartID (chart.id) is a metric, and we could actually have multiple charts rendered for the same metric - for example Dygraph and Gauge charts for "system.cpu". So to differentiate it i thought we could have different naming.

@VLegakis what do you think? This will be also a thing to remember when putting charts into Charts Table